src/CLI/Command/Check.php

```diff
@@ -177,6 +186,45 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         }
     }
 
+    /**
+     * @psalm-suppress UnresolvableInclude
+     */
+    protected function requireAutoload(?string $filePath, OutputInterface $output): void
```

Comment:
Why not invert params to have the nullable value at the end?

Reply:
Yup, I guess so